spot-client/src/spot-remote/ui/components/remote-control-menu/feedback-form.js

```diff
@@ -175,4 +211,38 @@ export default class FeedbackForm extends React.Component {
             </div>
         );
     }
+
+    /**
+     * Restarts the inactivity timeout.
+     *
+     * @private
+     * @returns {void}
+     */
+    _restartInactivityTimeout() {
+        this._cancelInactivityTimeout();
+        if (this.props.timeout && this.props.timeout > 0) {
+            this._timeout = setTimeout(
+                () => {
+                    this._submitFeedback(/* timeout */ true);
+                },
+                this.props.timeout);
+        }
+    }
+
+    /**
+     * Submits the feedback currently stored in the state (whatever it is).
+     *
+     * @param {boolean} timeout - Whether or not the feedback is being submitted as a result of
+     * the inactivity timeout.
+     * @private
+     * @returns {void}
+     */
+    _submitFeedback(timeout) {
+        logger.log(`Feedback submitting, timeout? ${timeout}`);
```

Comment:
It might be better to make the method simpler by not passing in a boolean and instead have separate logger statements at the call site of submitFeedback. I think it might be better because that removes conditional functionality of submitFeedback. I also think it doesn't really make much a difference but I'm gonna ramble on anyway.